### DISCUSSION TITLE
Windows support?

### DISCUSSION THREAD

**Discussion by jeffwitthuhn** [rxhanson/Rectangle#752] [Ideas]
This app makes window snapping on Mac OS even better than Windows. Any ideas on what it would take to get this working on Windows?

**Comment by rxhanson** [comment]
I don’t have plans to make a Windows version, but here is one made by someone else https://github.com/ahmetb/RectangleWin

**Reply by jeffwitthuhn** [reply]
Very cool, thanks for the link!
